plugins/woocommerce/src/Internal/Orders/OrderStatusController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Automattic\WooCommerce\Internal\Orders;
+
+use Automattic\WooCommerce\Internal\Traits\AccessiblePrivateMethods;
+use Automattic\WooCommerce\Internal\RestApiControllerBase;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Server;
+
+/**
+ * Controller for the REST endpoint to add order statuses to the WooCommerce REST API.
+ */
+class OrderStatusController extends RestApiControllerBase {
+	use AccessiblePrivateMethods;
```

Comment:
Since you started this PR, we've actually removed all usages of this trait, so this should be removed 😅 

See #52937

Reply:
fixed in https://github.com/woocommerce/woocommerce/pull/52455/commits/8ba33445e598dd11eefc212db7d24f6658fa3b53